pkg/restapi/edv/operation/operations.go

```diff
@@ -99,14 +101,15 @@ func (c *Operation) createDataVaultHandler(rw http.ResponseWriter, req *http.Req
 
 		_, err = rw.Write([]byte(fmt.Sprintf("Data vault creation failed: %s", err)))
 		if err != nil {
-			log.Errorf("Failed to write response for data vault creation failure: %s",
-				err.Error())
+			log.Errorf("Failed to write response for data vault creation failure: %s", err.Error())
```

Comment:
Not writing error to response? or sending error response code?

Reply:
This error is a result of the **responseWriter itself failing** - so the best I can do is log.

Reply:
(trying to write a response again would result in another error check...and then another error check for that error and so on and so forth)